src/character.cpp

```diff
@@ -5740,17 +5742,31 @@ void Character::mod_rad( int mod )
 
 int Character::get_stamina() const
 {
+    if( is_npc() ) {
+        // No point in doing a bunch of checks on NPCs for now since they can't use stamina.
+        return 10000;
```

Comment:
```suggestion
        return get_stamina_max();
```

This is more self-documenting and better than hard-coding the max for NPCs twice.